electrical/breakbeam.py

```diff
@@ -3,22 +3,25 @@
 from enum import Enum
 
 
+def ackerman_calculations (steer_ang, left_ang, right_ang, outside_ang, inside_ang, track_width, wheel_base, steering_ration):
+    pass
+
 class Breakbeam:
     ACCEPTABLE_TIME = 3  # time to wait to confirm a full break
 
-    class BEAM_PINS(Enum):
-        HALF1 = 17
-        HALF2 = 22
-        FULL1 = 23
-        FULL2 = 24
+    # class BEAM_PINS(Enum):
+    #     HALF1 = 17
+    #     HALF2 = 22
```

Comment:
You might want to remove this commented code if its not needed. It seems BEAM_PINS is in the class now so its not needed